### ISSUE TITLE
"react-arcgis": "^3.1.0" Can't resolve 'esri-promise'

### ISSUE THREAD

**Issue by greenhippie** [nicksenger/react-arcgis#39]
1st off - amazing library! I have been using 3.0.0 without issue.

I had to add: "esri-promise": "^0.4.1" to my package.json
otherwise I get the following error when trying to use 3.1.0
```
Module not found: Can't resolve 'esri-promise' in '<PROJECT_PATH>\node_modules\react-arcgis'
```
Looking @ the 3.1.0 index.js - it still contains:
```
!function(e,t){"object"==typeof exports&&"object"==typeof module?module.exports=t(require("react"),require("esri-promise"))
```
I'm not sure if you intended consumers of your package to have to explicitly depend on esri-promise, esp. when your 3.1.0 notes say you have moved away from it.

**Comment by nicksenger** [comment]
@greenhippie thanks for bringing this to my attention. It seems like there is definitely some module loading issue(s) with the current build of the package.

For now I'd suggest using version 3.0.0, as it is functionally equivalent to 3.1.0. I should have this fixed tomorrow some time in the AM (PST).

**Comment by nicksenger** [comment]
Should be fixed @3.1.1, but let me know if you are still having issues. I'll take a close look at this and #38 shortly.

**Comment by greenhippie** [comment]
Happy to report 3.1.1 resolves this issue 👍
